packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -111,19 +118,39 @@ const inputButton = style<PickerButtonProps | AriaSelectRenderProps>({
   alignItems: 'center',
   height: 'control',
   transition: 'default',
-  columnGap: 'text-to-control',
-  paddingX: 'edge-to-text',
+  columnGap: {
+    default: 'text-to-control',
+    isQuiet: 'text-to-visual'
+  },
+  paddingX: {
+    default: 'edge-to-text',
+    isQuiet: centerPadding()
+  },
   backgroundColor: {
     default: baseColor('gray-100'),
     isOpen: 'gray-200',
-    isDisabled: 'disabled'
+    isDisabled: 'disabled',
+    isQuiet: 'transparent'
   },
   color: {
     default: 'neutral',
     isDisabled: 'disabled'
   }
 });
 
+const quietFocusLine = style({
+  width: 'full',
+  // Use pixels since we are emulating a border.
+  height: '[2px]',
```

Comment:
should this really not be `size(2)`? Bit surprised that this focus line is an exception unlike our other focus rings but perhaps I'm misunderstanding when `size` should be used

Reply:
oh hmm im not sure...i just copied it over from another pr and im not sure if we used size() then. i'll take a closer look